tests/common.go

```diff
@@ -2789,7 +2789,12 @@ func CreateSourceAndDestClusters(orgID string, cloud_name string, uid string, ct
 	log.Infof("Save cluster %s kubeconfig to %s", SourceClusterName, srcClusterConfigPath)
 	err = CreateCluster(SourceClusterName, srcClusterConfigPath, orgID, cloud_name, uid, ctx)
 	if err != nil {
-		return err
+		if strings.Contains(err.Error(), "creation failed as it already exists") {
+			log.Errorf("%s already added for the user", SourceClusterName)
```

Comment:
you want to print this as an error or info. seems this error is expected

Reply:
Yes. This error is expected. I just want to print it out and move on and not fail the test because of it.